docs/features.md

```diff
@@ -291,16 +291,13 @@
 
 # Cost accounting
 ### As a node operator, I want to get the list of deploys and their costs when I run `show-blocks`.
-### As a node operator, I want to be compensated for storing, running, and processing transactions against smart contracts based on the computational intensity required.
-### As a node operator, I want to be compensated 
-### As a validator, I want to receive interest in phlo on my bond amount as defined by the schedule in the mint.
-### As a validator, having an absolute minimum phlo cost to deploy a contract helps support my resources and mitigate against DOS attacks.
-### As a validator, to support management of my resources I want to receive as a fee either a minimum charge for a deployment or a charge determined as a percentage of the size of the contract. The fee will be whichever is greater: the absolute minimum charge or the calculated minimum.
-### As a dApp developer, I use phlo to pay the cost to deploy smart contracts on the RChain platform.
-### As a dApp developer, I need to know the minimum cost it will take to deploy my contract. The cost will be whichever is greater: the absolute minimum charge or the calculated minimum.
-### As a dApp developer, I need to know how much it will cost to execute my contract.
-### As a dApp developer, I want the phlo cost to execute a contract a contract once it's on chain to be consistent given the asm REV to phlo price and accounting for non-determinism.
-### As a client of a contract already deployed on the blockchain, I need to know how much it will cost to execute the contract.
+### As a validator, I want to be compensated in REV for setting up, storing, running, and processing transactions. I need to be able to see a record of this compenstation and retrieve my earnings.
```

Comment:
validator -> validating node operator
Just to clarify the diff between what a human expects vs. what the software in the node expects.

Reply:
Fixed